app/app.js

```diff
@@ -215,16 +259,21 @@ function checkConnections() {
   const wasReady = state.ready;
   if (!state.shutdown) {
     const tasks = [dataConnection.checkConnection()];
+    if (featureFlags.eventStreamService) tasks.push(eventStreamService.checkConnection());
```

Comment:
The previous behaviour of `checkConnections` is that if the database is unavailable, the pods shut down. I'm not even sure if that is a good idea (cascading failures) but with the changes in this PR if anything goes wrong with NATS or the network, it looks like CHEFS will shut down, no? Since you've built retries into the event streaming, maybe CHEFS should stay up and reconnect if/when NATS becomes available?

Reply:
changed this for nats, will check the connection but application can run without a connection.